graph_altimeter/scan/iam/README.md

```diff
@@ -0,0 +1,30 @@
+# IAM Policies expansion
+
+This module recives the dictionary graph generated by
+[Altimeter](https://github.com/tableau/altimeter/blob/3bb875dff4a54aaf8df3c5dc38295a632259922f/altimeter/core/graph/graph_set.py#L125)
+and expands it to include an edge from each policy, embedded or not, to the
+edges representing the AWS resources the policies would potentially grant
+access to. It has the following limitations:
+
+* It does not ake into account the statements that "Deny" access to the resources,
+by now it only takes into account "Allow" statements.
+
+* Even though the edges it generates contain information about the type of
+access, `Read-Write` or only `Read`, that information is not actually persisted
+as the logic to persist custom properties in edges is not implemented yet by
+the Neptune client.
+
+* The conditions defined in the policies are not taken into account.
+
+* The assume role policies are not taken into account.
+
+## Determining the name of the AWS resources affected by an action
+
+The file: `iam_definition.json` contains the information about all the possible
+actions in a policy document and the names of the resources they affect to
+for a given AWS service.
+For instance, the action `DeleteTable` for the AWS Service: `dynamodb`
+affect to the resource `table`
+
+The file is copied from the utility: [parliament](https://github.com/duo-labs/parliament) and must be updated using
+the the tool defined in that repository.
```

Comment:
What tools do we have to use to update it? Should we include some instructions in this README?

Reply:
Yes I think it makes sense to detail this a little bit more.